src/challenges/RandomColorGenerator.tsx

```diff
@@ -1,15 +1,39 @@
-import { Container, Stack, Typography } from "@mui/material";
+import { Button, Container, Stack, Typography, Box } from "@mui/material";
 // Add required imports here
+import { useState } from "react";
 export default function RandomColorGenerator() {
-  // Add your code here
+  
+  const [color, setColor] = useState('#000000');
+
+  function hexa(){
+    let num;
+    const color = [];
+
+    for (let i = 0; i < 6; i++) {
+      num = Math.floor(Math.random()*10);
+      color.push(num);
+    }
+
```

Comment:
This function just gets hexadecimals with numbers. It does not use the whole range, please update to include every single hexadecimal possibility.

[Hexadecimal](https://www.varsitytutors.com/hotmath/hotmath_help/spanish/topics/hexadecimal-numbers)